src/main/resources/migrations/common/V2025.10.28__alter_reference_code.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE reference_code ADD COLUMN display_sequence SMALLINT;
+
+CREATE UNIQUE INDEX idx_reference_code_code_display_sequence ON reference_code (code, display_sequence);
```

Comment:
Does it need to be unique? And if so, should it include Group code, rather than item code? Nomis has loads of codes which are the same, in different groups  (or domains), and often has the display sequences either all 99 or all 1.